slsDetectorSoftware/src/Module.cpp

```diff
@@ -2614,6 +2614,11 @@ sls::IpAddr Module::getReceiverLastClientIP() const {
     return sendToReceiver<sls::IpAddr>(F_GET_LAST_RECEIVER_CLIENT_IP);
 }
 
+std::array<pid_t, NUM_RX_THREAD_IDS> Module::getReceiverThreadIds() const {
```

Comment:
It would be possible to return an sls::StaticVector if you only want to print the active threads. But it's fine to keep zeros as well